### PR TITLE
Add stSTARS information

### PR DESCRIPTION
**Pull request by vish-stride**
Add info+image for stSTARS, Stride's LSD for Stargaze's STARS token

### CODE REVIEW

File: packages/web/config/ibc-assets.ts [hunk 1]
```diff
@@ -868,6 +868,12 @@ export const IBCAssetInfos: (IBCAsset & {
     destChannelId: "channel-5",
     coinMinimalDenom: "stuatom",
   },
+  {
+    counterpartyChainId: "stride-1",
+    sourceChannelId: "channel-106",
```
Comment (JeremyParish69) [inline packages/web/config/ibc-assets.ts:873]: wrong src and dest channel Ids (these are for stargaze <-> stride), when it should be for (osmosis <-> stride)

**Comment by riley-stride** [comment]
Channels fixed, this is ready to merge I believe! @JeremyParish69

**Review by JeremyParish69 — APPROVED** [review]

File: packages/web/config/price.ts [hunk 2]
```diff
@@ -457,4 +457,17 @@ export const PoolPriceRoutes: IntermediateRoute[] = [
...
+    poolId: "810",
+    spotPriceSourceDenom: DenomHelper.ibcDenom(
+      [{ portId: "transfer", channelId: "channel-75" }],
+      "stustars"
+    ),
+    spotPriceDestDenom: DenomHelper.ibcDenom(
+      [{ portId: "transfer", channelId: "channel-0" }],
+      "ustars"
+    ),
+    destCoinId: "cosmos",
```
Comment (JeremyParish69) [inline packages/web/config/price.ts:471]: "cosmos" is ATOM's CGid. If opting for a CoingGecko ID, then use STARS' ("stargaze"). If opting for a local Osmosis spot price, then use "pool:ustars" (this gets the price from the STARS/OSMO pool)

File: packages/web/config/price.ts [hunk 3]
```diff
@@ -457,4 +457,17 @@ export const PoolPriceRoutes: IntermediateRoute[] = [
     spotPriceDestDenom: "uosmo",
     destCoinId: "osmosis",
   },
+  {
+    alternativeCoinId: "pool:stustars",
+    poolId: "810",
+    spotPriceSourceDenom: DenomHelper.ibcDenom(
+      [{ portId: "transfer", channelId: "channel-75" }],
```
Comment (JeremyParish69) [inline packages/web/config/price.ts:464]: should be channel-326

File: packages/web/config/price.ts [hunk 4]
```diff
@@ -457,4 +457,17 @@ export const PoolPriceRoutes: IntermediateRoute[] = [
...
+      "stustars"
+    ),
+    spotPriceDestDenom: DenomHelper.ibcDenom(
+      [{ portId: "transfer", channelId: "channel-0" }],
```
Comment (JeremyParish69) [inline packages/web/config/price.ts:468]: should be STARS' channel; i.e., channel-75

File: packages/web/config/price.ts (hunk 2 above)
Comment (vish-stride) [reply packages/web/config/price.ts]: fixed, thank you!

File: packages/web/config/price.ts (hunk 3 above)
Comment (vish-stride) [reply packages/web/config/price.ts]: fixed, thank you!

File: packages/web/config/price.ts (hunk 4 above)
Comment (vish-stride) [reply packages/web/config/price.ts]: fixed, thank you!

**Pull request merged**